packages/react-vapor/src/components/filterBox/FilterBox.tsx

```diff
@@ -102,6 +102,7 @@ export class FilterBox extends React.Component<IFilterBoxProps, any> {
                         onKeyUp={this.props.onKeyUp}
                         style={inputMaxWidth}
                         autoFocus={this.props.isAutoFocus}
+                        aria-label={`${this.props.id}-name`}

```

Comment:
The label should be something that labels the current element, here it's more of an id to identify the input?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

One thing to note is that you can get the input by its [placeholder with RTL](https://testing-library.com/docs/react-testing-library/cheatsheet/#queries) so I'm not sure we need that here